package.json

```diff
@@ -20,36 +20,42 @@
       "description": "Command or full path to pylint. Use %p for current project directory (no trailing /) or %h for current project name.",
       "order": 1
     },
+    "additionalArgs": {
+      "type": "string",
+      "default": "",
+      "description": "Additional pylint command arguments (e.g. '--load-plugins=pylint_django').",
+      "order": 2
+    },
     "pythonPath": {
       "type": "string",
       "default": "",
       "description": "Paths to be added to $PYTHONPATH. Use %p for current project directory or %f for the directory of the current file.",
-      "order": 1
+      "order": 3
```

Comment:
The `order` was being used as more of a priority, please leave the rest of these settings as they were. `2` is fine for the value of your new setting.